src/dbus/networkmanager/devices.rs

```diff
@@ -414,5 +414,5 @@ pub mod ip4 {
     }
 }
 pub use device::DeviceProxy;
-pub use statistics::StatisticsProxy;
+//pub use statistics::StatisticsProxy;
```

Comment:
Yeah you are right. Commented-out code does not look to professional. I've changed it now.
The intention was to leave a breadcrumb around in case someone wants to use `StatisticsProxy` in the future but it is not available.